apps/engineering/content/design/components/empty.example.tsx

```diff
@@ -0,0 +1,21 @@
+import { RenderComponentWithSnippet } from "@/app/components/render";
+import { BookBookmark, ShieldCheck } from "@unkey/icons";
+import { Button } from "@unkey/ui";
+import { Empty } from "@unkey/ui";
+
+export const EmptyExample: React.FC = () => (
+  <RenderComponentWithSnippet>
+    <Empty fill>
+      <Empty.Icon>
+        <ShieldCheck className="size-8" />
```

Comment:
The sizing should be handled automatically, otherwise the user may forget, like you did in some instances

The Empty.Icon should apply a class to its svg children like so `[&_svg]:size-9`, see the button implementation for reference